src/ResourceManager/Storage/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 * Upgrade SRP SDK to 7.1.0
```

Comment:
@blueww Please update the version of the module in AzureRM.Storage.psd1 (should be 4.1.0), please add the text here to the 'ReleaseNotes'  in AzureRM.Storage.psd1, and please update the AssemblyVersion and AssemblyFileVersion in AssemblyInfo.cs for the built assembly (Microsoft.Azure.Commands.Management.Storage).

Otherwise, this looks good.

Reply:
@markcowl  
All updated. Please check.